presto-main/src/main/java/com/facebook/presto/execution/QueryInfo.java

```diff
@@ -60,6 +60,8 @@
     private final String updateType;
     private final Optional<StageInfo> outputStage;
     private final FailureInfo failureInfo;
+    private final Optional<String> failureTask;
```

Comment:
@geraint0923 I thought we decided that these fields also go into FailureInfo? Did I miss some part of the discussion?